src/Async/Qu.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  Qu.swift
+//
+//  Created by Logan Wright on 10/24/15.
+//  Copyright © 2015 lowriDevs. All rights reserved.
+//
+
+public typealias Block = () -> ()
```

Comment:
Nope, typealiases are also namespaced, so if there's a conflicting `Block`, this one will be `IntrepidSwiftWisdom.Block`